cleanlab/object_detection/rank.py

```diff
@@ -192,13 +199,14 @@ def _compute_label_quality_scores(
 
     if method == "objectlab":
         scores = _get_subtype_label_quality_scores(
-            labels,
-            predictions,
+            labels=labels,
+            predictions=predictions,
             alpha=ALPHA,
             low_probability_threshold=LOW_PROBABILITY_THRESHOLD,
             high_probability_threshold=HIGH_PROBABILITY_THRESHOLD,
             temperature=TEMPERATURE,
             aggregation_weights=aggregation_weights,
+            overlapping_label_check=overlapping_label_check,
         )
```

Comment:
Not relevant to this PR but can you add an ValueError if the method is not `objectlab`?

Reply:
Added: https://github.com/cleanlab/cleanlab/blob/420e69bed2a1e64c6bf282b7da54359465361694/cleanlab/object_detection/rank.py#L212